src/log4net/Appender/RollingFileAppender.cs

```diff
@@ -1376,19 +1327,22 @@ protected void RollOverSize()
       CloseFile(); // keep windows happy.
 
       LogLog.Debug(declaringType, "rolling over count [" + ((CountingQuietTextWriter)QuietWriter).Count + "]");
-      LogLog.Debug(declaringType, "maxSizeRollBackups [" + m_maxSizeRollBackups + "]");
+      LogLog.Debug(declaringType, "maxSizeRollBackups [" + MaxSizeRollBackups + "]");
       LogLog.Debug(declaringType, "curSizeRollBackups [" + m_curSizeRollBackups + "]");
-      LogLog.Debug(declaringType, "countDirection [" + m_countDirection + "]");
+      LogLog.Debug(declaringType, "countDirection [" + CountDirection + "]");
 
-      RollOverRenameFiles(File);
+      if (File is not null)
+      {
+          RollOverRenameFiles(File);
```

Comment:
```suggestion
        RollOverRenameFiles(File);
```

Reply:
My 4-spaces settings were fighting the 2-space code. Added a .editorconfig in the next push.